validator/api/src/main/java/tech/pegasys/teku/validator/api/KeyStoreFilesLocator.java

```diff
@@ -80,12 +82,29 @@ private void parseEntry(final String keyFileName, final String passwordFileName)
               passwordFileName));
     }
     if (keyFile.isFile()) {
-      pathMap.putIfAbsent(keyFile.toPath(), passwordFile.toPath());
+      if (!isDepositDataFile(keyFile)) pathMap.putIfAbsent(keyFile.toPath(), passwordFile.toPath());
```

Comment:
nit: We always use braces on if statements:
```suggestion
      if (!isDepositDataFile(keyFile)) {
          pathMap.putIfAbsent(keyFile.toPath(), passwordFile.toPath());
      }
```